addons/binding/org.openhab.binding.elerotransmitterstick/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,26 @@
+Manifest-Version: 1.0
+Automatic-Module-Name: org.openhab.binding.elerotransmitterstick
+Bundle-ActivationPolicy: lazy
+Bundle-ClassPath: .,
+ lib/commons-codec-1.8.jar
```

Comment:
Please remove this lib from the bundle and import the package `org.apache.commons.codec[.*]` instead, which is available in the target platform.

Reply:
Done.

Reply:
Is it correct that the only place you need this is [here](https://github.com/openhab/openhab2-addons/pull/2659/files#diff-c13d5477e4054792862b03a9d4378a1cR55)?
If so, please use https://github.com/eclipse/smarthome/blob/master/bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/util/HexUtils.java instead.